src/main/java/controller/LottoController.java

```diff
@@ -0,0 +1,59 @@
+package controller;
+
+import model.Lotto;
+import view.InputView;
+import view.ResultView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoController {
+
+    private static final int TICKET_PRICE = 1000;
+    private final InputView inputView = new InputView();
+    private final ResultView resultView = new ResultView();
+
+    public void lottoRun() {
+        int purchaseAmount = parsePurchaseAmount(inputView.purchaseAmountTitle());
+        validatePurchaseAmount(purchaseAmount);
+
+        int lottoCount = purchaseAmount / TICKET_PRICE;
+
+        List<Lotto> lottoTickets = generateLottoTickets(lottoCount);
+        List<String> lottoTicketList = getLottoTicketStrings(lottoTickets);

```

Comment:
얼핏보기에 컬렉션의 자료형만 다른데

```suggestion
        List<Lotto> lottoTicketList = generateLottoTickets(lottoCount);
        List<String> lottoTickets = getLottoTicketStrings(lottoTickets);

```
변수명이 이렇게 바뀌어도 이상하지 않을 것 같아요
더 의미있게 네이밍을 수정해봅시다

Reply:
이번 스터디 통해서 네이밍 스킬을 더 길러보겠습니다 !
말씀하신대로의 리펙토링을 진행하였습니다 감사합니다 !